wagtail/snippets/templates/wagtailsnippets/snippets/create.html

```diff
@@ -1,43 +1,32 @@
-{% extends "wagtailadmin/base.html" %}
+{% extends "wagtailadmin/generic/create.html" %}
 {% load i18n wagtailadmin_tags %}
-{% block titletag %}{% blocktrans trimmed with snippet_type_name=model_opts.verbose_name %}New  {{ snippet_type_name }}{% endblocktrans %}{% endblock %}
-{% block bodyclass %}editor-view{% endblock %}
-{% block content %}
-    {% include 'wagtailadmin/shared/headers/slim_header.html' with breadcrumbs_items=breadcrumbs_items side_panels=side_panels history_url=history_url %}
-
-    {% trans "New" as new_str %}
-    {% include "wagtailadmin/shared/header.html" with title=new_str subtitle=model_opts.verbose_name icon=header_icon merged=1 only %}
-
-    <form action="{{ action_url }}" method="POST" novalidate{% if form.is_multipart %} enctype="multipart/form-data"{% endif %} data-edit-form>
-        {% csrf_token %}
-
-        <div class="nice-padding">
-            {{ panel.render_form_content }}
-        </div>
-
-        <footer class="footer">
-            <nav aria-label="{% trans 'Actions' %}">
-                <ul>
-                    <li class="actions actions--primary footer__container">
-                        <div class="dropdown dropup dropdown-button match-width">
-                            {{ action_menu.render_html }}
-                        </div>
-                    </li>
-                </ul>
-            </nav>
-        </footer>
-    </form>
 
+{% block slim_header %}
+    {% comment %}
+        We can't use the `only` keyword yet, because the SnippetStatusSidePanel still rely on some
+        context variables from the view. As a result, we can't reuse the include from the generic
+        template and have to copy it here without the `only` keyword.
+    {% endcomment %}
+    {% include 'wagtailadmin/shared/headers/slim_header.html' with breadcrumbs_items=breadcrumbs_items side_panels=side_panels history_url=history_url %}
 {% endblock %}
 
-{% block extra_css %}
-    {{ block.super }}
-    {{ media.css }}
+{% block actions %}
+    <footer class="footer footer--no-margin">
+        <nav aria-label="{% trans 'Actions' %}">
+            <ul>
+                <li class="actions actions--primary footer__container">
+                    <div class="dropdown dropup dropdown-button match-width">
+                        {{ action_menu.render_html }}
+                    </div>
+                </li>
+            </ul>
+        </nav>
+    </footer>
```

Comment:
Before merging, might want to wait until #9030 is implemented in case that work becomes hard to merge due to this change